preflight-checks/src/checks/02-check-port-and-healthcheck-route.js

```diff
@@ -15,6 +15,16 @@ const currentWorkingDirectory = process.cwd();
 const path = require( 'path' );
 const node_modules = path.join( currentWorkingDirectory, 'node_modules' );
 
+const envVariables = {
+	'VIP_GO_APP_ID': '123',
+}
+
+const executeShell = ( command, envVars = {} ) => {
+	return execa.command( command, {
+		env: Object.assign( envVariables, envVars )
```

Comment:
This will actually overwrite the global `envVariables`, polluting them for any other command that is called (they'll always have every command's env vars). Instead, we need to assign it all to a new object.
```suggestion
		env: Object.assign( {}, envVariables, envVars )
```

Reply:
Oh nice catch, thank you 🙏